src/Helpers/Conversations/FocusedOnConversationMessageInput.ts

```diff
@@ -0,0 +1,24 @@
+import { ActionType, FocusedOnConversationMessageInput } from "../../Schema"
+
+export interface FocusedOnConversationMessageInputArgs {
+  impulse_conversation_id: string
```

Comment:
We have been transforming casing to match our js consumers for these args, so would be `impulseConversationId`.

Reply:
Ohh nice catch! Fixed in 9afec93.